WebHostLib/static/assets/tutorial/ff1/multiworld_en.md

```diff
@@ -0,0 +1,114 @@
+# Final Fantasy 1 (NES) Multiworld Setup Guide
+
+## Required Software
+- The FF1Client which is bundled with [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases)
+- The [BizHawk](http://tasvideos.org/BizHawk.html) emulator. Versions 2.3.1 and higher are supported. 
+  Version 2.7 is recommended
+- Your Final Fantasy (USA Edition) ROM file, probably named `Final Fantasy (USA).nes`. Archipelago.gg nor the Final
```

Comment:
I think "Neither Archipelago nor the [...]" is better englissh, right?
(Not actually a native English speaker, so I definitely can be wrong.)

Reply:
I mean I am but I don't know either. "Neither" feels correct here